test/simple/test-net-pipe-connect-errors.js

```diff
@@ -32,6 +32,11 @@ var accessErrorFired = false;
 // Test if ENOTSOCK is fired when trying to connect to a file which is not
 // a socket.
 var emptyTxt = path.join(common.fixturesDir, 'empty.txt');
+if (process.env.NODE_PIPE_DIR !== undefined) {
```

Comment:
We might be able to always use:
```
emptyTxt = path.join(common.pipeTmpDir, 'empty.txt');
fs.closeSync(fs.openSync(emptyTxt, 'w'));
```
and never use the `empty.txt` file from the fixtures directory.